src/main/java/org/fcrepo/migration/handlers/ocfl/ArchiveGroupHandler.java

```diff
@@ -223,6 +233,46 @@ public void processObjectVersions(final Iterable<ObjectVersionReference> version
         handleDeletedResources(f6ObjectId, objectState, datastreamStates);
     }
 
+    private boolean fedora3DigestValid(final ContentDigest f3Digest) {
+        return f3Digest != null && !f3Digest.getType().isBlank() && !f3Digest.getDigest().isBlank();
```

Comment:
This is not safe if the type or digest are null. `StringUtils.isNotBlank()` should do the trick

Reply:
thanks, fixed